src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String name;
+    private int quantity;
+
+    public Fruit(String name) {
+        this.name = name;
+        this.quantity = 0;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public void addQuantity(int quantity) {
+        this.quantity += quantity;
+    }
+
+    public void subtractQuantity(int quantity) {
+        this.quantity -= quantity;
+    }
```

Comment:
When adding or subtracting quantity, it would be a good idea to include checks to ensure that the quantity does not become negative. For example, in the 'subtractQuantity' method, you should check if the 'quantity' after subtraction would be less than zero and handle it accordingly (e.g., throw an exception). This is to prevent cases where a negative quantity might be mistakenly set, which could lead to incorrect behavior or data inconsistencies.